backend/src/main/java/tqs/medex/service/AuthService.java

```diff
@@ -41,8 +42,11 @@ public JwtAuthenticationResponse authenticateUser(LoginRequest request) {
     SecurityContextHolder.getContext().setAuthentication(authentication);
 
     String jwt = tokenProvider.generateToken(authentication);
+    CustomUserDetails user = (CustomUserDetails) authentication.getPrincipal();
+    System.out.println(user);
```

Comment:
Use a logger instead

Reply:
That shouldn't be there anyway. I'll fix that